config.toml

```diff
@@ -14,7 +14,7 @@ contentDir = "content/en"
 defaultContentLanguage = "en"
 defaultContentLanguageInSubdir = false
 # Useful when translating.
-enableMissingTranslationPlaceholders = false
+enableMissingTranslationPlaceholders = true
```

Comment:
What's the reason behind this change?

Reply:
I introduced several new translated text blobs. This just flags all untranslated strings with the placeholder [i18n] identifier. It is easier to see if there is non-content stuff that is not localized. Can be turned off, but what I added is ready for multilanguage support.

Reply:
But I don't think we plan/want any multi-language support. Would already be good if we could have proper up to data and maintained docs in English :)

Reply:
I know. But I learned to do things right :)

Reply:
It was agreed to revert this.